### DISCUSSION TITLE
Set a fixed pixel resolution, and scale the pixels themselves with the window?

### DISCUSSION THREAD

**Discussion by iam-angel-val** [bevyengine/bevy#9709] [Q&A]
I'm trying to achieve an aesthetic with my game that replicates the aesthetic of 2D PS1/Dreamcast/Saturn games, and I believe a big part of that is having the game locked at 480p. I'd like to make it so that as the window is resized, even though the window itself is bigger, the content itself stays at that fixed height of 480 pixels (i'm not too fussed about the resolution width) and the pixels themselves are what is resized. I am not really familiar enough with the engine to even begin to know how to achieve this though. I've done some research and the closest thing to a solution I've seen is someone saying they rendered the game to a 480p slice of the window and then resized that to fit the entire window, but once again I don't really have any idea of how I'd do that.

**Comment by ilyas-taouaou** [comment]
You need to render your game to a texture with the resolution you want into a 2D full-screen quad mesh.
This example can be useful:
https://bevyengine.org/examples/3d-rendering/render-to-texture/
